packages/api/src/command/medical/document/document-query-storage-info.ts

```diff
@@ -22,12 +23,15 @@ export type S3Info = {
   fileContentType: string | undefined;
 };
 
-type SimpleFile = {
-  docId: string;
+// TODO we need to overhaul this whole file
```

Comment:
nit: what does this mean? perhaps we can be more descriptive here, tie it to an issue, or remove it entirely?

a TODO that's not actionable will result in us probably forgetting what it's for, and it not being useful